src/XrdCeph/XrdCephPosix.cc

```diff
@@ -706,8 +707,15 @@ int ceph_posix_open(XrdOucEnv* env, const char *pathname, int flags, mode_t mode
        logwrapper((char*)"Could not find size or stripe_unit xattr for %s", fr.name.c_str());
       }
      else{
-       fr.stripeUnit = std::stoull(d_stripeUnit.c_str());
-       fr.objectSize = std::stoull(d_objectSize.c_str());
+       char *cleanStripeUnit = new char[1024];
+       char *cleanObjectSize = new char[1024];
```

Comment:
If memory is dynamically allocated with `new`, it should be cleared somewhere.
Other option is to use static allocations.